smdebug/tensorflow/keras.py

```diff
@@ -109,7 +113,8 @@ def register_model(self, model):
         # It attaches a hook to every layer of the model to capture
         # layer values
         self.model = model
-        self._wrap_model_with_input_output_saver()
+        if self.tape is not None:
```

Comment:
why this change?

Reply:
The wrap model isn't needed with the new changes on TF to capture layer inputs and outputs when we use `.fit`